src/uwtools/cli/run_forecast.py

```diff
@@ -60,11 +70,23 @@ def parse_args(args: List[str]) -> Namespace:
         metavar="FILE",
     )
     optional.add_argument(
-        "-m",
-        "--machine",
-        help="Path to YAML platform-definition file",
+        "--forecast-app",
+        choices={"HAFS", "MRW", "SRW"},
+        help="The app to be run",
+    )
```

Comment:
Did we decide that the "app" concept wasn't applicable at the level of just running a forecast? If so, I think this argument could be removed.

Reply:
I had maintained this along with the model call due to having to parse the correct class to run, even though its not currently looking at the app for versioning. I could remove it at this stage for us to add back if we need it later?

Reply:
If we're not using it, YAGNI is probably the right philosophy.